src/Streamly/Data/Unicode/Stream.hs

```diff
@@ -70,13 +70,13 @@ module Streamly.Data.Unicode.Stream
     -- * Construction (Decoding)
       decodeLatin1
     , decodeUtf8
-    , decodeUtf8Lax
```

Comment:
This module is deprecated, do not change this. We only need to deprecate it. The same applies to `encodeLatin1Lax` and `encodeUtf8Lax` as well.

Reply:
Also, the behavior other routines in this module should perhaps remain the same as old.

Reply:
If we do this then there won't be any breaking changes, only deprecations.

Reply:
> Also, the behavior other routines in this module should perhaps remain the same as old.
Nah, can't do that it'd be completely unnecessary. 

Will deprecate the lax/lenient variants.